src/utils/constants.ts

```diff
@@ -1,5 +1,4 @@
 // Needed due to ESLint rule: @typescript-eslint/no-magic-numbers
 
 export const Zero = 0;
-export const One = 1;
```

Comment:
Heyo! I think we still need this `One`, because [it's imported elsewhere](https://github.com/cfpb/sbl-frontend/blob/285-filing-app-phase1/src/pages/ProfileForm/Step1Form/AssociatedFinancialInstitutions.tsx#L11)?

Reply:
:doh: Added back.  Thanks for catching that over zealous delete.